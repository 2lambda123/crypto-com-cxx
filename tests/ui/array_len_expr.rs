#[cxx::bridge]
mod ffi {
    struct Shared {
        arraystr: [String; "13"],
        arraysub: [String; 15 - 1],
    }
}

fn main() {}
