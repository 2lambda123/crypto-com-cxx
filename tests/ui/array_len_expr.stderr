error: array length must be an integer literal
 --> $DIR/array_len_expr.rs:4:28
  |
4 |         arraystr: [String; "13"],
  |                            ^^^^
